### ISSUE TITLE
Menu misplaced on admin interface

### ISSUE THREAD

**Issue by hachpai** [denkGroot/Spina#185]
Element nav#secondary is displayed horizontally instead of vertically. Element nav#primary is positioned above.

**Comment by Bramjetten** [comment]
Are you using the latest version of spina-template?

Verstuurd vanaf mijn iPhone

**Comment by hachpai** [comment]
Using spina-template 0.4.0
Using spina 0.8.3
I've tried in rails 4.2 and rails 5.0 and still the same...

**Comment by Bramjetten** [comment]
Try upgrading Spina to 0.10.0

**Comment by hachpai** [comment]
I get that:
```
Bundler could not find compatible versions for gem "activemodel":
  In snapshot (Gemfile.lock):
    activemodel (= 5.0.0.1)

  In Gemfile:
    rails (>= 5.0.0.1, ~> 5.0.0) was resolved to 5.0.0.1, which depends on
      activerecord (= 5.0.0.1) was resolved to 5.0.0.1, which depends on
        activemodel (= 5.0.0.1)

    rails (>= 5.0.0.1, ~> 5.0.0) was resolved to 5.0.0.1, which depends on
      activerecord (= 5.0.0.1) was resolved to 5.0.0.1, which depends on
        activemodel (= 5.0.0.1)

    spina (= 0.10.0) was resolved to 0.10.0, which depends on
      globalize was resolved to 3.0.4, which depends on
        activemodel (< 4.0.0, >= 3.0.0)
```

I had to add `gem 'breadcrumbs_on_rails', github: 'seuros/breadcrumbs_on_rails', branch: 'patch-1'
` before to make it work.

**Comment by hachpai** [comment]
It worked, by using
`
gem 'breadcrumbs_on_rails', github: 'seuros/breadcrumbs_on_rails', branch: 'patch-1'
gem 'globalize', github: 'globalize/globalize'
gem 'spina-template', github: 'denkGroot/Spina-template'
gem 'spina', github: 'denkGroot/Spina'
`
Thank you for you work!

PH
